### ISSUE TITLE
DisplayLinkProvider preferredFrameRateRange range error

### ISSUE THREAD

**Issue by kushsolitary** [jtrivedi/Wave#30]
Hi,

First of all - great library! I am trying this out with SwiftUI (although I think this issue is unrelated) but I am encountering an issue. On my physical device (iPhone 11 Pro), the returned value of `UIScreen.main.maximumFramesPerSecond` is 61. 

This makes the following error out:

```swift
        if #available(iOS 15.0, *) {
            let maximumFramesPerSecond = Float(UIScreen.main.maximumFramesPerSecond)
            let highFPSEnabled = maximumFramesPerSecond > 60
            let minimumFPS: Float = highFPSEnabled ? 80 : 60
            displayLinkProvider?.preferredFrameRateRange = .init(minimum: minimumFPS, maximum: maximumFramesPerSecond, preferred: maximumFramesPerSecond)
        }
```

Error:
> *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'invalid range (minimum: 80.00 maximum: 61.00 preferred: 61.00)'

For now, I am forking the repo to increase the `highFPSEnabled` threshold and see if that works. Is there any other way to make the check work more reliably?

**Comment by jtrivedi** [comment]
Interesting, good find! Let me take a look 👍

**Comment by jtrivedi** [comment]
Fixed with release `0.3.2`. Thanks for opening the issue!
